steps/bundle-universal-apk/0.0.3/step.yml

```diff
@@ -0,0 +1,82 @@
+title: Bundle Universal Apk
+summary: |
+  Generates an universal apk from an app bundle
+description: |
+  Runs a bundletool that gets an app bundle path and generates an universal apk that it can be upload to alternative app stores or test platforms.
+website: https://github.com/ch4vi/bitrise-step-bundle-universal-apk
+source_code_url: https://github.com/ch4vi/bitrise-step-bundle-universal-apk
+support_url: https://github.com/ch4vi/bitrise-step-bundle-universal-apk/issues
+published_at: 2019-02-04T12:02:54.817601+01:00
+source:
+  git: https://github.com/ch4vi/bitrise-step-bundle-universal-apk.git
+  commit: 7c7e986769f360b13524f036bc13fae0790a8dce
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- build
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: unzip
```

Comment:
`unzip`, `wget` and `curl` are all preinstalled, no need to specify them.

For a full reference: https://github.com/bitrise-io/bitrise.io/tree/master/system_reports

Declaring these will cause the build to run longer.

Reply:
In a previous PR, another reviewer rejected it because it did not include curl and unzip.
https://github.com/bitrise-io/bitrise-steplib/pull/1883#discussion_r250598889

Reply:
Yes, apologies, there was a misunderstanding on my side.

You have to list the dependencies you use at runtime, as discussed in the PR you referenced. Official Bitrise guidelines will be published soon, sorry for the inconvenience.

I'll approve this PR.